### ISSUE TITLE
Add Face Generation using Deep Learning

### ISSUE THREAD

**Issue by NEERAJAP2001** [akshitagupta15june/Face-X#536]
I'd like to add Face generation using deep learning as part of  [This](https://github.com/akshitagupta15june/Face-X/tree/master/Awesome-face-operations#face-generation-1) Section.

Contents :
* Pre-processed Data
* Create a DataLoader
* Define the Model
* Discriminator
* Generator
* Initialize the weights of your network
* Build a complete network
* Discriminator and Generator Losses
* Optimizers
* Training
* Training Loss
* Generator samples from the training

Dataset : [Here](https://www.kaggle.com/jessicali9530/celeba-dataset)

**Comment by akshitagupta15june** [comment]
Ok assigning

**Comment by NEERAJAP2001** [comment]
@akshitagupta15june this implementation should be in [This](https://github.com/akshitagupta15june/Face-X/tree/master/Awesome-face-operations) Folder right?

**Comment by akshitagupta15june** [comment]
Can you create a new folder inside awesome face operations by name face generation nd add your deep learning implementation in it as well as gan face generation which is also implemented inside that

**Comment by akshitagupta15june** [comment]
Means yours and gan face generation

**Comment by NEERAJAP2001** [comment]
@akshitagupta15june okay 👍
